smoothquant/calibration.py

```diff
@@ -57,11 +57,11 @@ def get_static_decoder_layer_scales(model,
                                     dataset_path,
                                     num_samples=512,
                                     seq_len=512,
-                                    return_all=False,
+                                    return_act_scales_dict=False,
```

Comment:
@Guangxuan-Xiao that name is now inconsistent with the argument used in export_int8_model.py
Could you fix that please?